### PR TITLE
Added release notes for 2.12 and 2.13

### PR DESCRIPTION
**Pull request by smahati**
**Description**

Changes proposed in this pull request:

- Release notes for 2.12 and 2.13

**Related issue(s)**

### CODE REVIEW

File: blog/2024-05-23-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+---
+
+You can read about the new features in Luigi v2.13 in the release notes below.
+
+<!-- Excerpt -->
+
+
+#### Accessibility for Left Nav in BTP Layout
+
+The Left Nav bar is now accessible when in BTP Layout.
```
[the remark this answers is not in the document]
Comment (JohannesDoberer) [reply blog/2024-05-23-release-notes.md:16]: @smahati
The change is that when the BTP layout is active, the left nav is accessible from the keyboard.

File: blog/2024-05-03-release-notes.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+<!-- Excerpt -->
+
+
+#### Opt Out of Luigi Error Handling for 404s
+
+If you have implemented your own `page not found` error handler function, then you can set **ignoreLuigiErrorHandling** to `true` in your routing configuration.
+
+#### Update Context via Core API
+
+This addition lets you update the context values for visible iframes and LUI web components.
```
Comment (JohannesDoberer) [inline blog/2024-05-03-release-notes.md:20]:
```suggestion
This addition lets you update the context values for visible iframes and Luigi web components.
```

File: blog/2024-05-03-release-notes.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+---
+
+You can read about the new features in Luigi v2.12 in the release notes below.
+
+<!-- Excerpt -->
+
+
+#### Opt Out of Luigi Error Handling for 404s
+
+If you have implemented your own `page not found` error handler function, then you can set **ignoreLuigiErrorHandling** to `true` in your routing configuration.
```
Comment (JohannesDoberer) [inline blog/2024-05-03-release-notes.md:16]:
```suggestion
If you have implemented your own `page not found` error handler function, then you can set [**ignoreLuigiErrorHandling**](https://docs.luigi-project.io/docs/navigation-parameters-reference?section=pagenotfoundhandler) to `true` in your routing configuration.
```

**Review by JohannesDoberer — APPROVED** [review]

File: blog/2024-05-03-release-notes.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+#### Opt Out of Luigi Error Handling for 404s
+
+If you have implemented your own `page not found` error handler function, then you can set [**ignoreLuigiErrorHandling**](https://docs.luigi-project.io/docs/navigation-parameters-reference?section=pagenotfoundhandler) to `true` in your routing configuration.
+
+#### Update Context via Core API
+
+This addition lets you update the context values for visible iframes and Luigi web components.
+
+#### Change Luigi Core Version on the Fly in Luigi Fiddle
```
Comment (hardl) [inline blog/2024-05-03-release-notes.md:22]: this is a feature in luigi-fiddle, not luigi itself, so should not be part of the luigi release notes

**Review by hardl — CHANGES_REQUESTED** [review]

**Review by hardl — APPROVED** [review]

**Pull request merged**